src/loader.ts

```diff
@@ -385,6 +389,40 @@ interface ModuleDefinitionArguments extends Array<any> {
 		pendingCacheInsert = {};
 	}
 
+	function loadNodeModule(moduleId: string, parent?: Module): any {
```

Comment:
no worries, will do the same for https://github.com/dojo/loader/pull/46/files#diff-1d670561cb1689a37382f052a97a2e97R908